.github/workflows/build-artifacts.yml

```diff
@@ -16,21 +18,24 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '3.1.x'
+          source-url: https://nuget.pkg.github.com/graphql-dotnet/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
       - name: Install dependencies
         working-directory: src
         run: dotnet restore
       - name: Build solution (release config)
         working-directory: src
-        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:VersionSuffix=${{ github.run_id }}
+        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:VersionSuffix=$(( 1000 + $GITHUB_RUN_NUMBER ))
```

Comment:
Why 1000+ ?

Reply:
Just to make it a 4 digit number. I don’t know enough about bash scripts to format a number with 4 digits.  We can change it.

Reply:
Is GITHUB_RUN_NUMBER a monotonically increasing counter? Can it go beyond the value 9999 (8999) and thus result will no longer be 4 digits? Why aim for a 4-digit number in version suffix at all?

Reply:
Yes, it is basically a run counter.

Yes, if we trigger the workflow > 9999 times, by making 10,000+ commits to master/develop.

I have no opinion - I simply chose that because appveyor created 4 digit numbers.  We don't have to.  The version number suffix would start at 1 (it's already at around 20 actually) and keep increasing.  So you'd get version numbers like 4.0.0.25, and eventually 4.0.0.103.  That's fine with me also.

Reply:
> I simply chose that because appveyor created 4 digit numbers

Appveyor started a 1, there have just been that many builds.  😉

Reply:
OK